### PR TITLE
Added target_category_column support  & Generation of inliers.csv, outliers.csv, scores.csv

### PR DESCRIPTION
**Pull request by govarsha**
Added code to support target_category_column in autots.
Updated AnomalyOutput class.
Changed report generation accordingly.
Added generation of inliers.csv, outliers.csv, scores.csv

Report
image
image

inliers.csv
image

outliers.csv
image

scores.csv
image

### CODE REVIEW

File: ads/opctl/operator/lowcode/anomaly/model/autots.py [hunk 1]
```diff
@@ -46,37 +46,65 @@ def _build_model(self) -> pd.DataFrame:
...
+                    inliers = df.loc[inlier_indices]
+                    scores = score
 
-            dataset.data[OutputColumns.ANOMALY_COL] = anomaly[col].values
+                    dataset.data[OutputColumns.ANOMALY_COL] = anomaly[col].values
 
-        else:
-            "TBD"
+            else:
+                "TBD"
```
Comment (ahosler) [inline ads/opctl/operator/lowcode/anomaly/model/autots.py:107]: Can we make this an error message?
Maybe:
raise NotImplementedError("Multi-Output Anomaly Detection is not yet supported in autots")

Comment (govarsha) [reply ads/opctl/operator/lowcode/anomaly/model/autots.py:101]: Yeah I will add it

File: ads/opctl/operator/lowcode/anomaly/model/autots.py [hunk 2]
```diff
@@ -46,37 +46,65 @@ def _build_model(self) -> pd.DataFrame:
...
-            self.anomaly_output = AnomalyOutput(
-                inliers=inliers, outliers=outliers, scores=scores
-            )
+                else:
+                    outliers = df.loc[outlier_indices]
+                    inliers = df.loc[inlier_indices]
+                    scores = score
 
-            dataset.data[OutputColumns.ANOMALY_COL] = anomaly[col].values
+                    dataset.data[OutputColumns.ANOMALY_COL] = anomaly[col].values
```
[the remark this answers is not in the document]
Comment (govarsha) [reply ads/opctl/operator/lowcode/anomaly/model/autots.py]: For now I have used full_data_dict to store both input and anomaly output and to plot. I will make the changes in AnomalyOutput as discussed and make changes accordingly.

File: ads/opctl/operator/lowcode/anomaly/model/autots.py (hunk 1 above)
Comment (govarsha) [reply ads/opctl/operator/lowcode/anomaly/model/autots.py]: Added

**Pull request merged**